### DISCUSSION TITLE
Error just from Windows using Apple Music app

### DISCUSSION THREAD

**Discussion by petcol** [mikebrady/shairport-sync#1868] [Q&A]
Hi all, Airplay has been working just fine from my Apple TV, my iPhone and also from iTunes on the PC previously.  iTunes is becoming redundant and you now need to install Apple Music App...  but I get an error message when trying to connect to the Raspberry Pi's running share-port sync
![2024-06-24 - Airplay from Apple Music off PC](https://github.com/mikebrady/shairport-sync/assets/61083722/656fd4a0-dfe1-48af-bb66-c92d3b8e0ebf)

Amy suggestions?

**Comment by mikebrady** [comment]
Thanks for the post. Shairport Sync will work with Apple Music on Windows if it is built for Classic operation. Unfortunately, it will not work in AirPlay 2 mode either with iTunes for Windows or Apple Music for Windows.
